app/views/home/index.html.erb

```diff
@@ -1 +1 @@
-<%= render partial: "signup_form" %>
+<div class="map"></div>
```

Comment:
Cool! I think that was the right move since we don't need the sign up form in the home view anymore, we shouldn't be testing it's there! 

Thanks for doing that!!